src/super_gradients/training/models/detection_models/customizable_detector.py

```diff
@@ -97,7 +102,7 @@ def _initialize_weights(self, bn_eps: Optional[float] = None, bn_momentum: Optio
     def prep_model_for_conversion(self, input_size: Optional[Union[tuple, list]] = None, **kwargs):
         for module in self.modules():
             if module != self and hasattr(module, "prep_model_for_conversion"):
-                module.prep_model_for_conversion(input_size, **kwargs)
+                module.prep_model_for_conversion(input_size, full_fusion=True, **kwargs)  # FIXME: Where should we set `full_fusion` ?
```

Comment:
What is the best way to set `full_fusion=True`?
If we set `full_fusion` to `True` by default this question would not be relevant. 

But If we don't I am not sure whether to hard code it here (might also create backward compatibility issue).
We could also just pass it in the kwargs. But it would be very likely that people will forget about it and even worse not even know about it.

So this makes me think that maybe we should just set it to `True` by default